eagles-food-backend/appsettings.json

```diff
@@ -1,6 +1,6 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "prod db connection string"
+    "DefaultConnection": ""
```

Comment:
hm does this matter? it still uses the development one anyway

Reply:
Okay. I'd return the test db connection string then